hexrd/material.py

```diff
@@ -348,6 +699,14 @@ def _set_atominfo(self, v):
 #
 
 
+xtal_sys_dict = {'cubic': 1,
+                 'tetragonal': 2,
+                 'orthorhombic': 3,
+                 'hexagonal': 4,
+                 'rhombohedral': 5,
```

Comment:
@saransh13 Should this one be "trigonal" instead of "rhombohedral"?

I currently get an error with the material.h5 file as follows:
```python
>>> from hexrd.material import Material
>>> Material('Gd_hp', 'material.h5').dump_material('out.h5')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/patrick/virtualenvs/hexrd/src/hexrd/hexrd/material.py", line 529, in dump_material
    AtomInfo['xtal_sys'] = xtal_sys_dict[self.unitcell.latticeType.lower()]
KeyError: 'trigonal'
```

Reply:
If so, I'll take care of fixing it.

Reply:
@psavery yep, should be trigonal. My bad.